### DISCUSSION TITLE
Prevent picom from adding transparency to client title

### DISCUSSION THREAD

**Discussion by vendion** [herbstluftwm/herbstluftwm#1519] [Q&A] [answered]
I'm trying to work out how to prevent picom from adding transparency to the title bar on clients, but no clue what the class to use in picom is for it. I've tried `_HERBST_TITLE` as my best guess, but that didn't work.

With picom running:
![picom-running](https://user-images.githubusercontent.com/145018/188727516-d56c3d06-4f61-46da-994a-72ff7f057a95.png)

Without picom:
![no-picom](https://user-images.githubusercontent.com/145018/188727592-bff6450f-a7bf-4307-ad29-ce87da68976f.png)

**Comment by t-wissmann** [comment, marked as answer]
```
$ xprop -frame
WM_CLASS(STRING) = "_HERBST_DECORATION", "_HERBST_DECORATION"
```

**Reply by vendion** [reply]
Ahh I was using `xprop` without the frame argument so I just kept getting the client window information. Thank you!
